src/validation/validateRequiredFences.ts

```diff
@@ -0,0 +1,120 @@
+import reportError from '../core/reportError';
```

Comment:
Can you run prettier on this file?  (Settings are in the .vscode/settings.json, if you use the VSCode plugin.)